wiki/Client/File_formats/osr_(file_format)/de.md

```diff
@@ -0,0 +1,101 @@
+# .osr (Dateiformat)
+
+**.osr** ist ein Dateiformat mit Informationen über ein osu!-Replay. Für die Verwendung muss sich die zum Replay gehörende Beatmap im Ordner "Songs" befinden.
+
+## Datentypen
+
+| Name | Länge in Bytes | Beschreibung |
+| :-- | :-- | :-- |
+| Byte | 1 | Ein einzelner 8-Bit-Wert. |
+| Short | 2 | Ein 2-Byte-Wert im Little-Endian-Format. |
+| Integer | 4 | Ein 4-Byte-Wert im Little-Endian-Format. |
+| Long | 8 | Ein 8-Byte-Wert im Little-Endian-Format. |
+| ULEB128 | Variabel | Eine Ganzzahl mit variabler Länge. Siehe [ULEB128](https://en.wikipedia.org/wiki/LEB128). |
+| String | Variabel | Besteht aus drei Teilen. Ein einzelnes Byte, das entweder 0x0b (11 im Dezimalsystem) oder 0x00 ist, abhängig davon, ob die nächsten zwei Teile vorhanden sind oder nicht. Bei einem Wert von 0x0b bestehen die nächsten zwei Teile aus einem ULEB128, der die Länge in Byte des folgenden Strings definiert, und der String selbst, kodiert in UTF-8. Siehe [UTF-8](https://de.wikipedia.org/wiki/UTF-8). |
+
+## Format
+
+Byte-Offsets sind in dieser Tabelle nicht enthalten, da die Werte unterschiedlich lang sein können.
+
+| Datentyp | Beschreibung |
+| :-- | :-- |
+| Byte | Spielmodus des Replays (0 = osu!, 1 = osu!taiko, 2 = osu!catch, 3 = osu!mania) |
```

Comment:
The original article still uses the old game mode names. I decided to go with the new ones since the French article also has them.